src/test/tasks/Xcode/1.0.2/xcodebuild.js

```diff
@@ -14,7 +14,7 @@ exports.execute = function(ctx, callback) {
 	var xcbpath = which('xcodebuild');
 	ctx.verbose('using xcodebuild: ' + xcbpath);
 
-	var repoPath = ctx.variables['build.sourceDirectory'];
+	var repoPath = ctx.variables['build.sourcesDirectory'];
```

Comment:
OK - find as hard coded string for now.  Task authors know via docs what the published strings are.